### PR TITLE
⭐️ add additional tags to policies

### PR DESCRIPTION
**Pull request by chris-rock**

### CODE REVIEW

File: core/mondoo-dns-security.mql.yaml [hunk 1]
```diff
@@ -5,6 +5,9 @@ policies:
     authors:
       - name: Mondoo, Inc
         email: hello@mondoo.com
+    tags:
+      mondoo.com/platform: dns
```
Comment (scottford-io) [inline core/mondoo-dns-security.mql.yaml:9]: would this be `host, dns` 🤔 

```suggestion
      mondoo.com/platform: host, dns
```

File: core/mondoo-github-security.mql.yaml [hunk 2]
```diff
@@ -5,6 +5,9 @@ policies:
     authors:
       - name: Mondoo, Inc
         email: hello@mondoo.com
+    tags:
+      mondoo.com/platform: github
```
Comment (scottford-io) [inline core/mondoo-github-security.mql.yaml:9]: Do we want to start calling out `saas` as a platform? 🤔

File: core/mondoo-gitlab-security.mql.yaml [hunk 3]
```diff
@@ -5,6 +5,9 @@ policies:
     authors:
       - name: Mondoo, Inc
         email: hello@mondoo.com
+    tags:
+      mondoo.com/platform: gitlab
```
Comment (scottford-io) [inline core/mondoo-gitlab-security.mql.yaml:9]: same thing here... `saas` ?

File: core/mondoo-linux-security.mql.yaml [hunk 4]
```diff
@@ -5,6 +5,9 @@ policies:
     authors:
       - name: Mondoo, Inc
         email: hello@mondoo.com
+    tags:
+      mondoo.com/platform: linux
```
Comment (scottford-io) [inline core/mondoo-linux-security.mql.yaml:9]: is this also a `host`? 🤔

File: core/mondoo-openssl-vulnerability.mql.yaml [hunk 5]
```diff
@@ -5,46 +5,49 @@ policies:
     authors:
       - name: Mondoo, Inc
         email: hello@mondoo.com
-    specs:
-      - asset_filter:
-          query: platform.family.contains(_ == 'unix')
-        scoring_queries:
-          mondoo-openssl-vulnerability:
+    tags:
+      mondoo.com/platform: linux,unix
```
Comment (scottford-io) [inline core/mondoo-openssl-vulnerability.mql.yaml:9]: `host` ? 🤔

File: core/mondoo-terraform-aws-security.mql.yaml [hunk 6]
```diff
@@ -5,6 +5,9 @@ policies:
     authors:
       - name: Mondoo, Inc
         email: hello@mondoo.com
+    tags:
+      mondoo.com/platform: aws,cloud
```
Comment (scottford-io) [inline core/mondoo-terraform-aws-security.mql.yaml:9]: `terraform`?

File: core/mondoo-windows-security.mql.yaml [hunk 7]
```diff
@@ -5,6 +5,9 @@ policies:
     authors:
       - name: Mondoo, Inc
         email: hello@mondoo.com
+    tags:
+      mondoo.com/platform: windows
```
Comment (scottford-io) [inline core/mondoo-windows-security.mql.yaml:9]: `host` ?

File: extra/mondoo-google-workspace-security.mql.yaml [hunk 8]
```diff
@@ -5,6 +5,9 @@ policies:
     authors:
       - name: Mondoo, Inc
         email: hello@mondoo.com
+    tags:
+      mondoo.com/platform: googleworkspace
```
Comment (scottford-io) [inline extra/mondoo-google-workspace-security.mql.yaml:9]: `saas` ?

File: extra/mondoo-slack-security.mql.yaml [hunk 9]
```diff
@@ -5,6 +5,9 @@ policies:
     authors:
       - name: Mondoo, Inc
         email: hello@mondoo.com
+    tags:
+      mondoo.com/platform: slack
```
Comment (scottford-io) [inline extra/mondoo-slack-security.mql.yaml:9]: `saas`?

**Review by scottford-io — APPROVED** [review]
@chris-rock A few comments that I will leave to you, but great additions!

File: core/mondoo-github-security.mql.yaml (hunk 2 above)
Comment (chris-rock) [reply core/mondoo-github-security.mql.yaml:9]: I was thinking about that too. Lets try

File: core/mondoo-terraform-aws-security.mql.yaml (hunk 6 above)
Comment (chris-rock) [reply core/mondoo-terraform-aws-security.mql.yaml:9]: +1

File: core/mondoo-dns-security.mql.yaml (hunk 1 above)
Comment (chris-rock) [reply core/mondoo-dns-security.mql.yaml:9]: I think host does not apply here

**Review by scottford-io — COMMENTED** [review]
Looks great!

**Pull request merged**